client/src/elm/Activity/Utils.elm

```diff
@@ -464,24 +464,8 @@ expectMotherActivity currentDate offlineSession motherId activity =
                                                         >> Maybe.withDefault False
                                                     )
                                                 |> LocalData.withDefault False
-
-                                        entitledByUbudehe =
-                                            Dict.get motherId offlineSession.mothers
-                                                |> Maybe.map
-                                                    (\mother ->
-                                                        case mother.ubudehe of
-                                                            Just Ubudehe1 ->
-                                                                True
-
-                                                            Just Ubudehe2 ->
-                                                                True
-
-                                                            _ ->
-                                                                False
-                                                    )
-                                                |> Maybe.withDefault False
                                     in
-                                    isBreastfeeding && entitledByUbudehe
+                                    isBreastfeeding
```

Comment:
@Diama1 now we don't need to have it as a param.
Remove the `let`, and apply logic directly.